ereg-mapper-core/src/main/java/no/nav/registre/ereg/mapper/EregMapper.java

```diff
@@ -341,13 +345,16 @@ private String createENH(
 
         String dateNowFormatted = getDateNowFormatted();
 
+        String undersakstype = endringsType.equals("E") ? "EN" : "NY";
```

Comment:
Kanskje gjøre "E", "EN", "NY" om til static final variabler for å gjøre det litt tydeligere hva de betyr?

Reply:
Tror jeg skal skrive om denne koden litt. Venter til da.